ui/eureka/eureka.js

```diff
@@ -2240,6 +2754,9 @@ UserConfig.getUserConfigLocation('eureka', Options, function(e) {
   addOverlayListener('onFateEvent', function(e) {
     gTracker.OnFate(e);
   });
+  addOverlayListener('onCEEvent', function(e) {
```

Comment:
This doesn't need to be a closure (the above listener as well),
```js
addOverlayListener('onCEEvent', gTracker.OnCE);
```

Reply:
Yep they can be condensed, thanks!

Reply:
On reflection I'm going to leave these to keep consistent with the other overlays and pre-existing code.